### ISSUE TITLE
Access multiple USB Cameras at same time

### ISSUE THREAD

**Issue by vidhyagithub** [saki4510t/UVCCamera#569]
Can the library access frames two USB Cameras concurrently?

**Comment by MauriDalga** [comment]
Yes u can, i do that.

**Comment by EjazAmir** [comment]
hey, can you share your project code where you use mulitple usb cameras? I'm really stuck in this

**Comment by MauriDalga** [comment]
I only did something very similar to 'usbCameraTest7'. Do you already see that example

**Comment by EjazAmir** [comment]
> I only did something very similar to 'usbCameraTest7'. Do you already see that example

Yeah, i tried but unable to compile the application. The error says, "No such class found". 
WIll you kindly share the piece of code that actually connects the mulitple cameras and captures images, not asking for whole project bcx i understand the confidentiality matters.
Regards

**Comment by EjazAmir** [comment]
> I only did something very similar to 'usbCameraTest7'. Do you already see that example

hey, i tried 'usbCameraTest7' but unable to capture images from multiple cameras, can you kindly help me that
